### PR TITLE
MP-7884 [BS-8103] NOX - Database configuration for "MacAddress"

### PR DESCRIPTION
**Pull request by emir-sehovic1**

### CODE REVIEW

File: src/Nox.Types.EntityFramework/Types/MacAddress/MacAddressDatabaseConfigurator.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    public void ConfigureEntityProperty(
+        NoxSolutionCodeGeneratorState noxSolutionCodeGeneratorState, 
+        EntityTypeBuilder builder, 
+        NoxSimpleTypeDefinition property, 
+        Entity entity, 
+        bool isKey)
+    {
+        builder
+            .Property(property.Name)
+            .IsRequired(isKey || property.IsRequired)
```
Comment (gjorgji-grgovski-authoritypartners) [inline src/Nox.Types.EntityFramework/Types/MacAddress/MacAddressDatabaseConfigurator.cs:24]: We don't need to check "isKey" anymore, based on discussion with @rochar, so please use only "property.IsRequired"

File: src/Nox.Types.EntityFramework/Types/MacAddress/MacAddressDatabaseConfigurator.cs [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+            .IsUnicode(false)
+            .IsFixedLength(true)
+            .HasMaxLength(12)
+            .HasConversion<MacAddressConverter>();
```
Comment (gjorgji-grgovski-authoritypartners) [inline src/Nox.Types.EntityFramework/Types/MacAddress/MacAddressDatabaseConfigurator.cs:28]: MacAddressConverter should use "FromDatabase"

**Review by gjorgji-grgovski-authoritypartners — CHANGES_REQUESTED** [review]

File: src/Nox.Types.EntityFramework/Types/MacAddress/MacAddressDatabaseConfigurator.cs (hunk 2 above)
Comment (emir-sehovic1) [reply src/Nox.Types.EntityFramework/Types/MacAddress/MacAddressDatabaseConfigurator.cs:28]: Fixed, thanks

File: src/Nox.Types.EntityFramework/Types/MacAddress/MacAddressDatabaseConfigurator.cs (hunk 1 above)
Comment (emir-sehovic1) [reply src/Nox.Types.EntityFramework/Types/MacAddress/MacAddressDatabaseConfigurator.cs]: Fixed, thanks

**Review by gjorgji-grgovski-authoritypartners — APPROVED** [review]

**Review by rochar — APPROVED** [review]

**Pull request merged**